src/fsharp/FSharp.Core/seq.fsi

```diff
@@ -164,6 +226,16 @@ namespace Microsoft.FSharp.Collections
         ///
         /// <returns>The result sequence.</returns>
         ///
+        /// <example>
+        /// <code lang="fsharp">
+        ///     type Foo = { Bar: int seq }
+        ///
+        ///     [[{Bar = [1; 2]}; {Bar = [3; 4]}] |> Seq.collect (fun foo -> foo.Bar) // evaluates to seq [1; 2; 3; 4]
+        ///     [[1; 2]; [3; 4]] |> Seq.collect id // evaluates to seq [1; 2; 3; 4]
+        ///     null |> Seq.collect (fun foo -> foo.Bar) // throws exception!
```

Comment:
I think don't give the `null` examples.  It's extremely rare to use `null` with `seq` and altough you can do it because it's a .NET type it's basically abnormal for F#, we don't need to highlight it here

Reply:
Removed